module/Display.js

```diff
@@ -1,22 +1,70 @@
-import Tasks from './Task.js';
+import LocalStorage from './LocalStorage.js';
 
 const listContainer = document.querySelector('.showList');
 export default class Display {
-  static addList() {
-    Tasks.forEach((task) => {
-      const div = document.createElement('div');
-      const input = document.createElement('input');
-      const p = document.createElement('p');
-      div.setAttribute('class', 'list');
-      input.setAttribute('type', 'checkbox');
-      input.setAttribute('class', 'mycheckbox');
-      p.setAttribute('class', 'displayP');
-      const pText = document.createTextNode(task.discription);
-      p.appendChild(pText);
-      div.appendChild(input);
-      div.appendChild(p);
-
-      listContainer.appendChild(div);
+  static addList(task, id) {
+    const div = document.createElement('div');
+    const div1 = document.createElement('div');
+    const input = document.createElement('input');
+    const input1 = document.createElement('input');
+    const button = document.createElement('button');
+    const i = document.createElement('i');
+    div.setAttribute('class', 'list');
+    input.setAttribute('type', 'checkbox');
+    input.setAttribute('class', 'mycheckbox');
+    input.setAttribute('id', id);
+    input1.setAttribute('class', 'displayP');
+    input1.setAttribute('value', task.des);
+    input1.setAttribute('type', 'text');
+    button.setAttribute('class', 'list-btn ');
+    i.setAttribute('class', 'fa-solid fa-trash');
+
+    button.appendChild(i);
+    div1.appendChild(input);
+    div1.appendChild(input1);
+    div.appendChild(div1);
+    div.appendChild(button);
+
+    listContainer.appendChild(div);
+
+    document.querySelector('.clearBtn').style.display = 'inline-block';
+  }
+
+  static renameLocal(id, e) {
+    const list = LocalStorage.getlist();
+    list.forEach((task, i) => {
+      if (task.id.toString() === id.toString()) {
+        list.splice(i, 1);
+        e.parentNode.parentNode.remove();
+      }
+    });
+    Display.changeindex(list);
+
+    if ((Object.keys(list).length) === 0) document.querySelector('.clearBtn').style.display = 'none';
+    localStorage.setItem('list', JSON.stringify(list));
+  }
+
+  static changebool(id) {
```

Comment:
- [ ] Kindly ensure when a user changes the task to complete the change should be stored in local storage, I currently can't see a completed task when I save it.

- [ ] Also am getting this error when I run your app,  kindly check the configuration of your webpack:

![image](https://user-images.githubusercontent.com/61408860/184180858-e66979f4-f9dc-46d6-9f96-ae1f71b7f629.png)